DigitalLearningSolutions.Data.Tests/Services/RecommendedLearningServiceTests.cs

```diff
@@ -491,7 +527,33 @@ private void GivenResourceForSelfAssessmentIsReturnedByLearningHubApi(decimal ra
             A.CallTo(
                 () => competencyLearningResourcesDataService.GetCompetencyLearningResourcesByCompetencyId(CompetencyId)
             ).Returns(new List<CompetencyLearningResource> { competencyLearningResource });
+        }
+
+        private void GivenResourceHasTwoCompetencies()
+        {
+            A.CallTo(() => selfAssessmentDataService.GetCompetencyIdsForSelfAssessment(SelfAssessmentId))
+                .Returns(new[] { CompetencyId, 3 });
+
+            var competencyLearningResources = Builder<CompetencyLearningResource>.CreateListOfSize(2)
+                .All()
+                .With(clr => clr.LearningResourceReferenceId = LearningResourceReferenceId)
+                .And(clr => clr.LearningHubResourceReferenceId = LearningHubResourceReferenceId)
+                .And(clr => clr.AdminId = 7)
+                .TheFirst(1)
+                .With(clr => clr.Id = CompetencyLearningResourceId)
+                .And(clr => clr.CompetencyId = CompetencyId)
+                .TheRest()
+                .With(clr => clr.Id = 2)
+                .And(clr => clr.CompetencyId = 3)

```

Comment:
Seeing how we have the first Clr.Id and clr.CompetencyId defined as consts, it would be good to define the second ones as consts as well to make the tests that use it a bit clearer.